src/store/flow/block-types/Core_SetGroupMembershipStore.ts

```diff
@@ -41,8 +41,19 @@ const actions: ActionTree<IEmptyState, IRootState> = {
     }, {root: true})
   },
 
-  async validate(_context, {block, schemaVersion}: {block: IBlock, schemaVersion: string}) {
-    return Core_SetGroupMembershipValidator.runAllValidations(block, schemaVersion)
+  async validateWithProgrammaticLogic(
```

Comment:
@object-required / @bulatgab: I'm avoiding to mix the usage of Class validator & validation on Vuex actions. I'm keeping the vuex approach because that's easy to override and to implement.

The pbm of the previous 
```
async validate(_context, {block, schemaVersion}: {block: IBlock, schemaVersion: string}) {
    return Core_SetGroupMembershipValidator.runAllValidations(block, schemaVersion)
}
```
is that we need to override that again in voto5, so we need to duplicate many codes. Besides, I found it hard to debug when we customize things in voto5.

Reply:
See the implication in voto5 side https://bitbucket.org/voto/voto5/pull-requests/2646/feature-vmo-6623-fix-validation-extension#Lresources/js/floip/builder/stores/block-types/Viamo_Core_SetContactPropertyBlockStore.tsF20

Reply:
@object-required , I've created a chart summary to explain the pbm quickly, see https://miro.com/app/board/o9J_lXulmu4=/?moveToWidget=3458764531173515492&cot=14